sci-libs/libgeodecomp/files/libgeodecomp-0.5.0_pre20200926-hpx.patch

```diff
@@ -0,0 +1,75 @@
+From f447f51e6a9e256df6439640290ed67132b2956b Mon Sep 17 00:00:00 2001
```

Comment:
Is the patch upstreamed?

Reply:
Usually I upstream patches from to time. But, seems like upstream worked on this also **and** hpx 1.5.1. has been released in the mean time...

I'll update the PR. Thanks!